crates/auth-nats-account/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "wasmcloud-auth-natsaccount"
+version = "0.1.0"
+authors = ["wasmCloud Team"]
+edition = "2018"
+description = "Authorizer plugin for wasmCloud that consults a NATS account server"
+
+[dependencies]
+reqwest =  {version = "0.11.3", features = ["blocking"] }
+wasmcloud-host = { path = "../wasmcloud-host"}
```

Comment:
Will we ever release this authorizer on crates.io? If so, this will need the version information for `wasmcloud-host`